javascript/javascript1/week2/student-manager/student.js

```diff
@@ -0,0 +1,34 @@
+const class07Students = ['Chris', 'Anne', 'Colin'];
+
+ function addStudentToClass(studentName) {
+     if(studentName===""){
+        return "Cannot be empty student name!!!"
+    }
+     if(class07Students.length<6){
+          for(let i=0;i<class07Students.length;i++)
+         {
+            if(class07Students[i].includes(studentName))
+            {
+             console.log(`student ${studentName} already exits`); 
+             return;
+            }
+        } 
+        class07Students.push(studentName);
+        
+    } else if(studentName==="queen"){
+        class07Students.push(studentName);
+    } else{
+        return "Cannot add more students to class 07";
+    } 
+    return class07Students;
+}
+
+function getNumberOfStudents() {
+  // You write code here
+   return class07Students.length; 
+} 
+console.log(addStudentToClass("Michael"));
+console.log(addStudentToClass("queen"));
+console.log(addStudentToClass("Julia"));
+
+console.log(getNumberOfStudents());
```

Comment:
Nice! remember that you can console.log different stages of your function and use template literals to fully show how your function works. 💪